### ISSUE TITLE
A11y_Visual Studio Code Service_Download_Download VS Code_Usability: Visually "See SHA-256 Hashes" button looks like a link.

### ISSUE THREAD

**Issue by Yogi2204** [microsoft/vscode-docs#6222]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #A11yUsable; #Closed; 

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 112.0.5615.49 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Download" button and press enter.
3. Tab till "See SHA-256 Hashes" button.
4. Verify whether the See SHA-256 Hashes button visually also looks like button or not.

### Actual Result
Visually "See SHA-256 Hashes" button looks like a link.

### Expected Result
Visually "See SHA-256 Hashes" button should be looks like a button only.

### User Impact
User will not able to differentiate the "See SHA-256 Hashes" control as button or link. 

### Attachment
![Bug_Usability](https://user-images.githubusercontent.com/130456851/231436800-abfb113f-0c39-4803-a0c0-933747fc8ef9.gif)

**Comment by gregvanl** [comment]
Duplicate of https://github.com/microsoft/vscode-docs/issues/5546 for context

**Comment by Yogi2204** [comment]
As per the comment in Duplicate bug 
"If there is a button and it is visually appearing as link control then it is NAB as long as screen reader should read the correct role for it, visually they can appear as anything".
Hence closing this bug.
